README.md

```diff
@@ -1760,6 +1760,8 @@ The following extractors use this feature:
   * e.g. `studio.youtube.com`, `youtubei.googleapis.com`
   * Note: Cookies exported from `www.youtube.com` will not work with hosts other than `*.youtube.com`
 * `innertube_key`: Innertube API key to use for all API requests
+* `lang`: Supported language code by YouTube to prefer translated fields of this language, if available (case-sensitive).
+  * See youtube.py for list of supported codes.
```

Comment:
Give a permalink to the relevant code

Reply:
actually, nvm - any link you create won't be in master. I'll generate permalink after merging the PR